test/node/node_test.cpp

```diff
@@ -320,7 +348,8 @@ TEST(NodeTest, StdArrayWrongSize) {
 }
 
 TEST(NodeTest, StdVector) {
-  std::vector<int> primes;
```

Comment:
Can you leave these tests as-is, and then create duplicate test cases with the custom allocator?